inc/display.php

```diff
@@ -66,8 +66,27 @@ function createBoardlist($mod=false) {
     foreach ($xboards as $val) {
         $boards[$val['uri']] = $val['title'];
     }
+    $body = '';
+    if (isset($config['prepended_foreign_boards'])){
```

Comment:
You shouldn't copy paste this block, please refactor this out to it's own function.